sources/sqlserver/toddl.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package sqlserver handles schema and data migrations from sqlserver.
+package sqlserver
+
+import (
+	"github.com/cloudspannerecosystem/harbourbridge/common/constants"
+	"github.com/cloudspannerecosystem/harbourbridge/internal"
+	"github.com/cloudspannerecosystem/harbourbridge/schema"
+	"github.com/cloudspannerecosystem/harbourbridge/spanner/ddl"
+)
+
+const (
+	stringLimit int64 = 2621440
```

Comment:
Instead of redefining the limit here, consider defining a var StringMaxLength along with the spanner vars and using that value here. We already do it for [PGMaxLength](https://github.com/cloudspannerecosystem/harbourbridge/blob/master/spanner/ddl/ast.go#L69)

Reply:
Note: We use StringMaxLength only to differentiate between an integer length and using `MAX`. We still assign `ddl.MaxLength` if the source length is > StringMaxLength (2621440)

Reply:
Right, removed the constant from the current package and moved it to the ddl package.